src/main/resources/builtin/builtin.gobra

```diff
@@ -13,8 +13,19 @@ type any = interface{}
 type error interface {
 	pred ErrorMem()
 
+	ghost
 	requires acc(ErrorMem(), _)
 	decreases
+	pure IsDuplicableMem() bool
+
+	ghost
+	preserves ErrorMem()
```

Comment:
Given that `IsDuplicableMem` takes a wildcard permission amount, it might make sense to add a `perm` parameter to `Duplicate`. Otherwise, `IsDuplicableMem`'s contract could be changed to take full permission

Reply:
I would rather not complicate the signature of `Duplicate` with a perm ammount. Furthermore, I think that this will still result in the same performance degradation as we currently have with the wildcard version. Regarding the second suggestion, if I understand it correctly, any permission to a predicate amount in a pure function's precondition is now encoded as a wildcard - as such, there is no point in changing the syntax to something that is interpreted in the same way